diff --git a/YUView.pro b/YUView.pro
index 2ed8e70d..548b33bf 100644
--- a/YUView.pro
+++ b/YUView.pro
@@ -1,9 +1,7 @@
 TEMPLATE = subdirs
-SUBDIRS = YUViewLib YUViewApp YUViewUnitTest
+SUBDIRS = YUViewLib YUViewApp
 
 YUViewApp.subdir = YUViewApp
 YUViewLib.subdir = YUViewLib
-YUViewUnitTest.subdir = YUViewUnitTest
 
 YUViewApp.depends = YUViewLib
-YUViewUnitTest.depends = YUViewLib
diff --git a/YUViewApp/YUViewApp.pro b/YUViewApp/YUViewApp.pro
index 2f1d5742..05b41ce9 100644
--- a/YUViewApp/YUViewApp.pro
+++ b/YUViewApp/YUViewApp.pro
@@ -32,7 +32,6 @@ unix:!mac {
     icon128.files = $$top_srcdir/packaging/linux/icons/128x128/de.rwth_aachen.ient.YUView.png
     icon256.files = $$top_srcdir/packaging/linux/icons/256x256/de.rwth_aachen.ient.YUView.png
     icon512.files = $$top_srcdir/packaging/linux/icons/512x512/de.rwth_aachen.ient.YUView.png
-    icon1024.files = $$top_srcdir/packaging/linux/icons/1024x1024/de.rwth_aachen.ient.YUView.png
     icon32.path = $$PREFIX/share/icons/hicolor/32x32/apps
     icon64.path = $$PREFIX/share/icons/hicolor/64x64/apps
     icon128.path = $$PREFIX/share/icons/hicolor/128x128/apps
